frappe/public/js/frappe/views/workspace/blocks/paragraph.js

```diff
@@ -75,7 +75,7 @@ export default class Paragraph extends Block {
 	}
 
 	open_block_list() {
-		let dropdown_title = "Templates";
+		let dropdown_title = __("Templates");
 		let $block_list_container = $(`
 			<div class="block-list-container dropdown-list">
 				<div class="dropdown-title">${dropdown_title.toUpperCase()}</div>
```

Comment:
we convert the `dropdown_title` to upper case over here.

`toUpperCase` doesn't work properly with different languages (ref: https://stackoverflow.com/a/30763402)

Reply:
That toUpperCase is no my change, I just adding the text inside __( ) function because is not translatable now